content/docs/command-reference/run.md

```diff
@@ -5,6 +5,8 @@ command.
 
 ## Synopsis
 
+# TODO logs
+
```

Comment:
Is this a hypothetical section to explain logs? Would be an H3 under **Description**, prob 🙂